keystorage/interlock/src/main/java/tech/pegasys/signers/interlock/InterlockSessionFactoryProvider.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2020 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.signers.interlock;
+
+import tech.pegasys.signers.interlock.vertx.InterlockSessionFactoryImpl;
+
+import java.nio.file.Path;
+
+import io.vertx.core.Vertx;
+import org.apache.commons.lang3.StringUtils;
+
+public class InterlockSessionFactoryProvider {
+  private static final String HTTP_CLIENT_TIMEOUT_ENV = "INTERLOCK_CLIENT_TIMEOUT";
```

Comment:
nit: make sure this gets into the documentation

Reply:
added a note in github issue for documentation.